Use AUTOBOOT_KEYED to stop boot with Enter key only

This prevents garbage on the UART lines from interrupting the normal
boot process in U-Boot.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Index: u-boot-1.3.4/include/configs/at91sam9m10g45ek.h
===================================================================
--- u-boot-1.3.4.orig/include/configs/at91sam9m10g45ek.h
+++ u-boot-1.3.4/include/configs/at91sam9m10g45ek.h
@@ -207,6 +207,12 @@
 #define CFG_LONGHELP		1
 #define CONFIG_CMDLINE_EDITING	1
 
+#define CONFIG_AUTOBOOT_KEYED           /* Enable password protection   */
+#define CONFIG_AUTOBOOT_PROMPT          \
+        "\nBooting... stop with ENTER\n"
+#define CONFIG_AUTOBOOT_DELAY_STR       "\r"    /* 1st "password"       */
+#define CONFIG_AUTOBOOT_DELAY_STR2      "\n"    /* 2nd "password"       */
+
 #define ROUND(A, B)		(((A) + (B)) & ~((B) - 1))
 /*
  * Size of malloc() pool
